diff --git a/package/base-files/Makefile b/package/base-files/Makefile
index 17b6e40..330c381 100644
--- a/package/base-files/Makefile
+++ b/package/base-files/Makefile
@@ -41,7 +41,7 @@ define Package/base-files
   CATEGORY:=Base system
   DEPENDS:= \
 	+netifd +libc +jsonfilter +SIGNED_PACKAGES:usign +SIGNED_PACKAGES:openwrt-keyring \
-	+NAND_SUPPORT:ubi-utils +fstools +fwtool \
+	+NAND_SUPPORT:ubi-utils \
 	+SELINUX:procd-selinux +!SELINUX:procd +USE_SECCOMP:procd-seccomp \
 	+SELINUX:busybox-selinux +!SELINUX:busybox \
 	+ucode +ucode-mod-fs +ucode-mod-digest
